dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkApplicationListener.java

```diff
@@ -84,7 +85,7 @@ else if (event.getType() == RequestEvent.Type.RESP_FILTERS_START) {
                         commitTransaction();
                     } catch (Exception e) {
                         rollbackTransaction();
-                        this.<RuntimeException>rethrow(e);
+                        throw new MappableException(e);
```

Comment:
* I beleive Jersey already takes care of propogating an original exception, if a mapping has not been found. [Source](https://github.com/jersey/jersey/blob/master/core-server/src/main/java/org/glassfish/jersey/server/ServerRuntime.java#L612-619).
* Do we really have checked exceptions? `commitTransaction` doesn't throw any.